requirements.txt

```diff
@@ -180,10 +180,10 @@ django-debug-toolbar==3.2.4 ; python_version >= "3.9" and python_version < "4.0"
 django-oauth-toolkit==2.2.0 ; python_version >= "3.9" and python_version < "4.0" \
     --hash=sha256:46890decb24a34e2a5382debeaf7752e50d90b7a11716cf2a9fd067097ec0963 \
     --hash=sha256:abd85c74af525a62365ec2049113e73a2ff8b46ef906e7104a7ba968ef02a11d
-django-picklefield==3.1 ; python_version >= "3.9" and python_version < "4.0" \
+django-picklefield==3.1 ; python_version >= "3.9" and python_version < "4" \
```

Comment:
There are some additional/unrelated Python requirements being added here, but I think that's okay. Most are just restricting to < Python 4 (which I guess is coming at some point?).